<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Confirm</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Style sheet for the buttons */
QPushButton {
	color: rgb(76, 76, 76);
	background-color: rgb(235, 235, 235);
	border: 1px solid rgb(153, 153, 153);
	border-radius: 4px;
	min-width: 60px;
	min-height: 20px;
}

QPushButton:pressed {
    background-color: rgb(204, 204, 204);
}

QPushButton:default {
	border-color: rgb(76, 76, 76);
}

/*Style sheet for other elements*/

QLineEdit {
	border: 1px solid rgb(153, 153, 153);
	border-radius: 4px;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>222</width>
     <height>69</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_confirm">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="button_delete">
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(235, 76, 76);
	color:  rgb(235, 235, 235);
	border-color: rgb(76, 76, 76);
}

QPushButton:pressed {
    background-color: rgb(76, 76, 76);
}</string>
        </property>
        <property name="text">
         <string>Delete Preset</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>button_delete</tabstop>
  <tabstop>button_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
